### ISSUE TITLE
MyAnimeList Sync Support

### ISSUE THREAD

**Issue by Kyusetzu** [miru-project/miru-app#37]
I'd love to scroll through my library through this app and have the status of any series sync with MAL.

**Comment by HiroDane** [comment]
Also, same with Anilist and Kitsu will be nice. Full two-way sync.

**Comment by AxharKhan** [comment]
Yes please, I would like this app to sync my watch like [Aniyomi](https://github.com/aniyomiorg/aniyomi-preview) and [MalSync](https://github.com/MALSync/MALSync) do.

**Comment by Fenway17** [comment]
Yes pls add this feature
